src/tiledimage.js

```diff
@@ -533,6 +549,32 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
                 OpenSeadragon.getElementPosition( this.viewer.element ));
     },
 
+    // private
+    // Convert rectangle in tiled image coordinates to viewport coordinates.
+    _tiledImageToViewportRectangle: function(rect) {
+        var scale = this._scaleSpring.current.value;
```

Comment:
Here we are converting between the point coordinate of the tiled image (x between 0 and 1) to the viewport coordinates (taking scale and rotations into account).